src/pal/src/safecrt/safecrt_output_s.c

```diff
@@ -41,4 +41,5 @@ typedef char        _TCHAR;
 typedef char        TCHAR;
 #define _T(x)       x
 
+#include <stdarg.h>
```

Comment:
OK!

My intention was to show you that code using `wchar_t`/`char` is `typedef`ed later. I will do it.